docs/fastddsgen/interfaces/includes/interfaces.rst

```diff
@@ -0,0 +1,87 @@
+.. _fastddsgen_interfaces_definition:
+
+Defining an IDL interface
```

Comment:
Add another section for data streaming (@feed operations) describing the data streaming from both client and server point of view.